### PR TITLE
Setup QA

### PR DESCRIPTION
**Pull request by drautureau-sonarsource**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -40,114 +39,25 @@
...
-    <sonar.pluginName>Android</sonar.pluginName>
-    <sonar.requirePlugins>java:${sonar-java.version}</sonar.requirePlugins>
     <lint.version>22.2.0</lint.version>
     <simplexml.version>2.7.1</simplexml.version>
     <jsr305.version>3.0.0</jsr305.version>
     <slf4j.version>1.7.10</slf4j.version>
     <commons-io.version>2.4</commons-io.version>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
+    <!-- license headers -->
+    <license.owner>SonarSource SA and Jerome Van Der Linden, Stephane Nicolas, Florian Roncari, Thomas Bores</license.owner>
```
Comment (jeromevdl) [inline pom.xml]: Thanks ! :-)

File: its/plugin/pom.xml [hunk 2]
```diff
@@ -3,14 +3,13 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.sonarsource.parent</groupId>
-    <artifactId>parent</artifactId>
-    <version>23</version>
+    <groupId>org.sonarsource.android</groupId>
+    <artifactId>sonar-android</artifactId>
+    <version>1.2-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
```
Comment (benzonico) [inline its/plugin/pom.xml]: Why is that required ?

Comment (drautureau-sonarsource) [inline its/plugin/pom.xml]: By default it looks the parent to `../pom.xm` on the file system (which is not the case there), otherwise it looks in the local repo (but we don't use the `install` goal), and lastly in the remote repo (but it has not be deployed yet) : http://maven.apache.org/ref/3.0/maven-model/maven.html#parent
